src/Microsoft.Dnx.Runtime/ProjectResolver.cs

```diff
@@ -12,7 +12,7 @@ public static string ResolveRootDirectory(string projectPath)
         {
             var di = new DirectoryInfo(projectPath);
 
-            while (di.Parent != null)
+            while (di.Exists && di.Parent != null)
```

Comment:
I think it was necessary to allow the tests to provide a bogus project directory without exploding everything. It might not be necessary now that I'm calling a smaller surface area. I'll double check